src/types/addressPlugin.ts

```diff
@@ -3,6 +3,7 @@ import { OnEvents } from 'yavent'
 export interface PluginEvents {
   connect: undefined
   disconnect: undefined
+  error: unknown
```

Comment:
We shouldn't leak these out. Instead, we should do `console.log("Plugin error:", cleanUrl, error)` or something like that. If we notice a lot of re-occurring log entries, we can maybe turn these errors into metrics or whatever, but the point is that the hub doesn't need to deal with this.

Reply:
By lifting the concern we can test for it though. The only trade off is that logging would be done at a higher level in the stack.